recipe/meta.yaml

```
@@ -66,24 +94,69 @@ outputs:
         - darts.utils
         - darts.utils.data
         - darts.utils.historical_forecasts
+      requires:
+        - pip
+        - testfixtures
+        - pytest
       source_files:
         - darts
       commands:
-        - pytest darts/tests
+        - pip check
+        # - pytest -vvv -s darts/tests
+
+  - name: u8darts
+    skip: true  # [py<38 or s390x]
+    requirements:
+      build:
+        - patch     # [not win]
+        - m2-patch  # [win]
```

It's redundant for this one and all outputs below
```suggestion
      build:
        - patch     # [not win]
        - m2-patch  # [win]
```
It's redundant for this one and all outputs below